ci/py3.8.yml

```diff
@@ -26,7 +26,7 @@ dependencies:
   - numpy
   - pandas
   - pip
-  - prefect
+  - prefect>=1.3.0
```

Comment:
I think `>=1.3.0` will bring in the latest Prefect, so we probably just want `==`?